### ISSUE TITLE
pal generate creates a lot more files 

### ISSUE THREAD

**Issue by ralvs** [paljs/prisma-tools#119]
Hey guys.
I'm using version 1.0.3 and today the command `pal g` starts creating lots of files like follow
...MaxAggregateOutputType
...MinAggregateOutputType
...SumAggregateOutputType
...AvgAggregateOutputType

How can I stop that?

My pal.js config is
```
module.exports = {
  backend: {
    generator: 'sdl',
    output: 'graphql/generated',
  },
}
```

**Comment by AhmedElywa** [comment]
Yes it’s because new features in new version 2.2.0 I will upgrade now to feat new version of Prisma

**Comment by ralvs** [comment]
Ok, @AhmedElywa , understood.
So I think the problem is, I did not enable the experimental flag for aggregations

```
generator client {
  provider = "prisma-client-js"
  // experimentalFeatures = ["aggregateApi"]
}
```

And when Paljs creates all these types, I get a lot of erros like unknown type

`Unknown type "ExpenseMaxAggregateOutputTypeWhereInput". Did you mean "ExpenseMaxAggregateOutputType", "ExpenseMinAggregateOutputType", "ExpenseAvgAggregateOutputType", "ExpenseSumAggregateOutputType", or "NfeMaxAggregateOutputType"?`

**Comment by AhmedElywa** [comment]
@ralvs thanks for your report please use the last version of @paljs/cli v1.0.6 will fix this issue

**Comment by ralvs** [comment]
Great @AhmedElywa !
I have fixed that downgrading to prisma v2.1.3 
Will try update prisma and paljs now.
Thanks

**Comment by AhmedElywa** [comment]
@ralvs [v1.1.1](https://github.com/paljs/prisma-tools/releases/tag/v1.1.1) support adding `aggregate`
